.travis.yml

```diff
@@ -23,10 +23,16 @@ script:
       cd /coala; python3 -m pytest;
       cd /coala-bears;
       rm bears/Constants.py;  # There are no tests covering this module
-      rm bears/c_languages/CSharpLintBear.py tests/c_languages/CSharpLintBearTest.py;
-      rm bears/java/InferBear.py tests/java/InferBearTest.py;
-      rm bears/haskell/GhcModBear.py tests/haskell/GhcModBearTest.py;
-      rm -r bears/verilog tests/verilog/;
+      rm
+      bears/c_languages/CSharpLintBear.py
```

Comment:
Can you please add indentation for arguments , so that it is easy to see where the next command starts.